bot/seasons/easter/egghead_quiz.py

```diff
@@ -0,0 +1,118 @@
+import asyncio
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'egghead_questions.json'), 'r', encoding="utf8") as f:
+    EGGHEAD_QUESTIONS = load(f)
+
+
+EMOJIS = [
+    '\U0001f1e6', '\U0001f1e7', '\U0001f1e8', '\U0001f1e9', '\U0001f1ea',
+    '\U0001f1eb', '\U0001f1ec', '\U0001f1ed', '\U0001f1ee', '\U0001f1ef',
+    '\U0001f1f0', '\U0001f1f1', '\U0001f1f2', '\U0001f1f3', '\U0001f1f4',
+    '\U0001f1f5', '\U0001f1f6', '\U0001f1f7', '\U0001f1f8', '\U0001f1f9',
+    '\U0001f1fa', '\U0001f1fb', '\U0001f1fc', '\U0001f1fd', '\U0001f1fe',
+    '\U0001f1ff'
+]
+
+
+class EggheadQuiz(commands.Cog):
+    """This cog contains the command for the Easter quiz!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.quiz_messages = {}
+
+    @commands.command(aliases=["eggheadquiz", "easterquiz"])
+    async def eggquiz(self, ctx):
+        """
+        Gives a random quiz question, waits 30 seconds and then outputs the answer
+
+        Also informs of the percentages and votes of each option
+        """
+
+        random_question = random.choice(EGGHEAD_QUESTIONS)
+        question, answers = random_question["question"], random_question["answers"]
+        answers = [(EMOJIS[i], a) for i, a in enumerate(answers)]
+        correct = EMOJIS[random_question["correct_answer"]-1]
+
+        valid_emojis = [emoji for emoji, _ in answers]
+
+        description = "\n".join([f"{emoji} -> **{answer}**" for emoji, answer in answers])
```

Comment:
It would be helpful to include the time limit somewhere in the question. Define this as a constant so you can also use it in the `asyncio.sleep()` call.